es-ds-docs/pages/molecules/slider.vue

```diff
@@ -0,0 +1,70 @@
+<script setup>
+const model = ref(null);
+const min = ref(0);
+const max = ref(1000);
+const step = ref(50);
+const ariaLabel = ref('Sample aria-label');
+
+const propTableRows = [
+    ['starting-value', 'Number', '1', 'Required. Initial value of slider.'],
+    ['min', 'Number', '1', 'Required. Minimum value of slider.'],
+    ['max', 'Number', '100', 'Required. Maximum value of slider.'],
+    ['step', 'Number', '1', 'Step factor to increment/decrement the value.'],
+    ['aria-label', 'String', '"Pick a number"', 'Aria-label for slider handle.'],
+    ['label-formatter', 'Function', 'n/a', 'Function that modifies label value.'],
+];
+
+const { $prism } = useNuxtApp();
+const compCode = ref('');
+const docCode = ref('');
+
+onMounted(async () => {
+    if ($prism) {
+        /* eslint-disable import/no-webpack-loader-syntax, import/no-self-import */
+        const compSource = await import('@energysage/es-ds-components/components/es-slider.vue?raw');
+        const docSource = await import('./slider.vue?raw');
```

Comment:
Just needs one comment above this line `// eslint-disable-next-line import/no-self-import`